xml/cap_depl_install_caasp.xml

```diff
@@ -406,15 +433,55 @@ persistentvolumeclaim "test-sc-persistent" deleted</screen>
   </para>
 
   <para>
-   The example <filename>scf-config-values.yaml</filename> file is for a simple
-   deployment without an ingress controller or load balancer. Instead, assign
-   one worker node an external IP address, and map this to the domain name to
-   provide external access to the cluster. The <literal>external_ips</literal>
-   needs this address to provide access to the Stratos Web interface.
-   <literal>external_ips</literal> also needs the internal IP addresses of the
-   worker nodes to provide access to services.
+   The example configuration file <filename>scf-config-values.yaml</filename>
+   is for a simple deployment without the following network resources that are
+   usually relevant for production systems:
   </para>
+  <itemizedlist>
+   <listitem>
+    <formalpara>
+     <title>Hardware Load Balancer</title>
+     <para>
+      If you are using a hardware load balancer, assign one of the worker
```

Comment:
Not sure this is right-- hardware load balancers don't map to a single worker node AFAIK. That's one of the reasons to have them.

Reply:
@tgeek77 Can you answer this one?

Reply:
I think ideally you wouldn't want the load balancer to just one worker. You would want it to be able to use all workers.

The current text is:

>  Instead, assign one worker node an external IP address, and map this to the domain name to provide external access to the cluster.

That's confusing. A better way would be to say:

If you are using a hardware load balancer, assign at least one or more of the workers...

Reply:
Leave this as is for now and I'll clean it up in a separate commit, as it is a bit confusing. We've omitted hardware load balancers on purpose from this chapter, and plan to document them later.